### ISSUE TITLE
Bug? Found `NullPointerException` in `PhaseVerilog.scala`

### ISSUE THREAD

**Issue by tanhongze** [SpinalHDL/SpinalHDL#805]
When I replace my RAM with a black boxer, I get a `NullPointerException` in `PhaseVerilog.scala`.
```
case bb: BlackBox if bb != null => {
       val str = bb.impl.getVerilog()
    ...
}
```
Here, `bb` is not null but `bb.impl` is null. How to fix it?
And is there any way to fix it in my code before next Spinal release?

**Comment by tanhongze** [comment]
@Dolu1990 ?

**Comment by Dolu1990** [comment]
Hi ^^

Sorry, i was gone in holiday, AFK :D

Yes you are right, it was recently fixed : 
https://github.com/SpinalHDL/SpinalHDL/blob/1514ec64d47b0efe7c73ea69d846c0b51330924f/core/src/main/scala/spinal/core/internals/PhaseVerilog.scala#L106

and released, in SpinalHDL 1.7.2
